app/views/static_pages/index/_teen_raffle.html.erb

```diff
@@ -20,8 +20,7 @@
           on <%= Date.current.end_of_month.strftime("%B %e, %Y") %>.
           <% if already_joined %>
             <span class="italic block mt-2 text-sm">
-              You have already joined the raffle this month. Remember to check
-              back in <%= next_month %> for the next Raffle!
+              You have already joined the raffle this month.
```

Comment:
Let's keep the existing copy so they know they can come back next month. I think we're basically guaranteed to always have monthly raffles at this point.